### PR TITLE
Auth/feature/39775/remove required zip for international

### PR DESCRIPTION
**Pull request by mdewey**
## Description
- Removed the requirement for international zip code 
- behind a feature toggle
- Added factory method for feature toggle 

## Original issue(s)
[department-of-veterans-affairs/va.gov-team#0000
](https://github.com/department-of-veterans-affairs/va.gov-team/issues/39775)

## Testing done
- Added e2e tests

## Screenshots
Screen Shot 2022-04-13 at 3 51 45 PM

## Notes
- The BE is not ready for this. 
- The [feature toggle for the BE](https://github.com/department-of-veterans-affairs/vets-api/pull/9624/files)

### CODE REVIEW

File: src/applications/personalization/profile/mocks/feature-toggles/index.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+const generateFeatureToggles = (toggles = {}) => {
+  const {
+    profileDoNotRequireInternationalZipCode = true,
```
Comment (mdewey) [inline src/applications/personalization/profile/mocks/feature-toggles/index.js:3]: Values are taken from staging today

File: src/applications/personalization/profile/tests/e2e/address-validation/page-objects/AddressPage.js [hunk 2]
```diff
@@ -2,11 +2,11 @@
 import { setUp } from '@@profile/tests/e2e/address-validation/setup';
 
 class AddressPage {
-  loadPage(config) {
+  loadPage = config => {
```
Comment (mdewey) [inline src/applications/personalization/profile/tests/e2e/address-validation/page-objects/AddressPage.js:5]: Only linting changes.

File: src/applications/personalization/profile/components/ProfileInformationEditView.jsx [hunk 3]
```diff
@@ -222,6 +224,10 @@ export class ProfileInformationEditView extends Component {
       value,
       schema,
       uiSchema,
+      {
+        doNotRequireInternationalPostalCode: this.props
+          .doNotRequireInternationalPostalCode,
```
Comment (adamwhitlock1) [inline src/applications/personalization/profile/components/ProfileInformationEditView.jsx:229]: Looks like we need to add this to the prop types.

**Review by adamwhitlock1 — APPROVED** [review]
LGTM

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Pull request merged**